### DISCUSSION TITLE
Flux source reconcilation interval

### DISCUSSION THREAD

**Discussion by mo-saeed** [fluxcd/flux2#5040] [General]
Hello,

in the recommended settings here https://fluxcd.io/flux/components/kustomize/kustomizations/#recommended-settings I see the source reconcilation interval is 1m and i would like to ask if this small interval has any performance impact on flux, I am using OCI/ECR and s3 bucket.

**Comment by stefanprodan** [comment]
It really depends on the rate limits the upstream service has. Flux does not download the content every minute, it queries the upstream service and if the Git SHA, or S3 checksum or OCI digest changes from the last downloaded one, only then Flux will perform the new download.
